cmd/odf/get/dr_prereq.go

```diff
@@ -0,0 +1,109 @@
+package get
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+	submarinerv1alpha1 "github.com/submariner-io/submariner-operator/api/v1alpha1"
+	submarinerv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/cidr"
+	"golang.org/x/net/context"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const submarinerOperatorNamespace string = "submariner-operator"
+
+var drPrereqCmd = &cobra.Command{
+	Use:                "dr-prereq",
+	Short:              "Print the status of pre-requisites for Disaster Recovery between peer clusters.",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+	Example:            "odf get dr-prereq <peer-clusterID>",
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
@umangachapagain can we move implementation details of `Run:` to under `pkg/` folder to make `cmd/` more clean? Thanks

Reply:
Moved.